Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    qry_prov.connect(WorkspaceConfig(workspace="Cent"), mp_az_auth="cli")
</pre><p>Hard-coded workspace name used here.</p><p>Maybe would be better to do the workspace selection below first, then run this cell using the chosen workspace name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>